lerna.json

```diff
@@ -0,0 +1,14 @@
+{
+  "packages": [
+    "packages/*"
+  ],
+  "version": "independent",
```

Comment:
@alexnm we need to a dummy git publish in ``independent`` mode before adopting, today I was experimenting with ``independent`` mode and lerna pushed all packages as individual releases in my fork.

Reply:
what do you mean by git publish? npm?

Reply:
sorry, i mean Releases tab in github :sweat_smile: